src/napari_copick/widget.py

```diff
@@ -24,9 +24,13 @@
 
 
 class CopickPlugin(QWidget):
-    def __init__(self, viewer, config_path=None):
+    def __init__(self, viewer=None, config_path=None):
         super().__init__()
-        self.viewer = viewer
+        if viewer:
+            self.viewer = viewer
+        else:
+            self.viewer = napari.Viewer()
+
```

Comment:
Retains the ability to call from shell, but also allows starting from within Napari.